src/displayInfo.js

```diff
@@ -0,0 +1,22 @@
+const country = document.querySelector('.country-name');
+const countryFlag = document.getElementById('country-flag');
+const countryImg = document.querySelector('#country-image');
+const date = document.getElementById('date');
+const weatherIcon = document.getElementById('weather-icon');
+const description = document.querySelector('.description');
+const temperature = document.getElementById('temperature');
+const feels = document.getElementById('feels-like');
+
+
+const displayInfo = (results) => {
```

Comment:
- **_use object destructuring to get the values from an object. This way you can avoid repeating a lot of code. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/destructuring_requirement.png) from Pathwright) for example, you can do something like this: `displayInfo = ({weather, main, sys, name}) => {...}`_**